db/migrate/20250226001048_add_search_to_overriden_robots.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class AddSearchToOverridenRobots < ActiveRecord::Migration[7.2]
+  def up
+    googlebot_agent_disallow =
+      "User-agent: Googlebot\nDisallow: /admin/\nDisallow: /auth/\nDisallow: /assets/browser-update*.js\nDisallow: /email/\nDisallow: /session"
+    googlebot_agent_disallow_search_added =
+      "User-agent: Googlebot\nDisallow: /admin/\nDisallow: /auth/\nDisallow: /assets/browser-update*.js\nDisallow: /email/\nDisallow: /search\nDisallow: /session"
+    if select_value(
+         "SELECT value FROM site_settings WHERE name = 'overridden_robots_txt' AND value LIKE '%#{googlebot_agent_disallow}%'",
+       )
+      execute <<~SQL
+        UPDATE site_settings
+        SET value = REPLACE(value, '#{googlebot_agent_disallow}', '#{googlebot_agent_disallow_search_added}')
+        WHERE name = 'overridden_robots_txt'
+      SQL
+    end
+  end
+
+  def down
+    # raise ActiveRecord::IrreversibleMigration
```

Comment:
Need to uncomment?

Reply:
Hah, I copied it from #30782, I assumed @nattsw put it like that to indicate that it was okay to not raise the exception. 🙂

Reply:
Unless it's reversible we should raise it, having it here commented out is confusing